framework/assets/yii.validation.js

```diff
@@ -170,6 +155,10 @@ yii.validation = (function ($) {
                 }
             });
 
+            if (options.not === undefined) {
+                options.not = false;
```

Comment:
We expect client validators to behave the same way as according server validators. By default `not` is `false`, but if it was set to undefined, we get wrong result.